<launch>
        <param name="/side_state" type="int" value="1" />
        <param name="/mission_waitTime" type="double" value="1" />
        <param name="/file_name_mission" type="string" value="missionPoint.csv" />
        <param name="/file_name_path" type="string" value="scriptSmall.csv" />
        <param name="/feedback_activate" type="bool" value="false" />
        <rosparam param="/missionTime_correct_Type">[]</rosparam>
        <rosparam param="/missionTime_correct_Num">[]</rosparam>
        <rosparam param="/set_Chassis_Param_Type">[]</rosparam>
        <rosparam param="/set_Chassis_Param_xy_tolerance">[]</rosparam>
        <rosparam param="/set_Chassis_Param_theta_tolerance">[]</rosparam>
    <node name="Main_Node_Smol" pkg="main_2022_smol" type="main_2022_smol" output="screen">
    <!-- <node name="Main_Node_Smol" pkg="main_2022_smol" type="main_2022_smol_Emerge" output="screen"> -->
    </node>
</launch>